lib/vdsm/storage/lvm.py

```diff
@@ -577,9 +579,18 @@ def _reload_single_pv(self, pv_name):
 
         with self._lock:
             if error:
-                return self._update_stale_pvs_locked([pv_name])
+                updated_pvs = self._update_stale_pvs_locked([pv_name])
+                if pv_name not in updated_pvs:
+                    raise se.InaccessiblePhysDev((pv_name,))
```

Comment:
Shouldn't we include the error in se.InaccessiblePhysDev like se.VolumeGruopDoesNotExists?

This will let the caller log the error if needed. What we do here will always hide the lvm error.